legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/core/FunctionExecutionSupport.java

```diff
@@ -113,7 +117,9 @@ public interface FunctionExecutionSupport
     String GENERATE_EXECUTION_PLAN_ID = "legend.executionPlan.generate";
     String GRAMMAR_TO_JSON_LAMBDA_ID = "legend.grammarToJson.lambda";
     String JSON_TO_GRAMMAR_LAMBDA_BATCH_ID = "legend.jsonToGrammar.lambda.batch";
+    String GRAMMAR_TO_JSON_VALUE_SPECIFICATION_BATCH_ID = "legend.grammarToJson.valueSpecification.batch";
     String GET_LAMBDA_RETURN_TYPE_ID = "legend.lambda.returnType";
+    String GET_LAMBDA_RELATION_TYPE_ID = "legend.lambda.relationType";
```

Comment:
Why we need this for?  Seems redundant to the call above - GET_LAMBDA_RETURN_TYPE_ID?

Reply:
It's different functionality. the `V1_GraphManagerEngine` implementation on the client side has a `getLambdaReturnType` function which uses the `GET_LAMBDA_RETURN_TYPE_ID` and a `getLambdaRelationTypeFromRawInput` function which uses the `GET_LAMBDA_RELATION_TYPE_ID`. Similar names but they call different engine functions under the hood.

Reply:
But to me these are the same things... relation is just a type that a lambda/function can return.  Seems duplicated to me.  I rather align to have a single return type API, and if you need the type, get the UI to extract it, etc

Reply:
Let me experiment some and see if I can do that. Perhaps one lambda return type function, and if the return type is a relation, we also get the relation type and return them both in the response.

Reply:
Updated the code to just have the `getLambdaReturnType` function, and if the return type is a relation, then we also add the relation type to the response object.